roles/openshift_health_checker/test/podlogs_test.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+import json
+
+
+from openshift_checks.logging.podlogs import PodLogQueryTime, OpenShiftCheckException
+
+
+SAMPLE_UID = "3ea53ca5-f60a-40e5-8828-983240aee8a1"
+
+
+def canned_podlogquerytime(exec_oc=None):
+    """Create a check object with a canned exec_oc method"""
+    check = PodLogQueryTime("dummy")  # fails if a module is actually invoked
+    if exec_oc:
+        check.oc_cmd = exec_oc
+    return check
+
+
+def assert_error(error, expect_error):
+    if expect_error:
+        assert error
+        assert expect_error in error
+    else:
+        assert not error
+
+
+plain_running_elasticsearch_pod = {
+    "metadata": {
+        "labels": {"component": "es", "deploymentconfig": "logging-es-data-master"},
+        "name": "logging-es-data-master-1",
+    },
+    "status": {
+        "containerStatuses": [{"ready": True}, {"ready": True}],
+        "phase": "Running",
+    }
+}
+plain_running_kibana_pod = {
+    "metadata": {
+        "labels": {"component": "kibana", "deploymentconfig": "logging-kibana"},
+        "name": "logging-kibana-1",
+    },
+    "status": {
+        "containerStatuses": [{"ready": True}, {"ready": True}],
+        "phase": "Running",
+    }
+}
+not_running_kibana_pod = {
+    "metadata": {
+        "labels": {"component": "kibana", "deploymentconfig": "logging-kibana"},
+        "name": "logging-kibana-2",
+    },
+    "status": {
+        "containerStatuses": [{"ready": True}, {"ready": False}],
+        "conditions": [{"status": "True", "type": "Ready"}],
+        "phase": "pending",
+    }
+}
+
+
+@pytest.mark.parametrize('pods, expect_pods', [
+    (
+        [not_running_kibana_pod],
+        [],
+    ),
+    (
+        [plain_running_kibana_pod],
+        [plain_running_kibana_pod],
+    ),
+    (
+        [],
+        [],
+    )
+])
+def test_check_running_pods(pods, expect_pods):
+    check = canned_podlogquerytime(None)
+    pods = check.running_pods(pods)
+    assert pods == expect_pods
+
+
+@pytest.mark.parametrize('json_response, uuid, timeout, expect_exception, extra_words', [
+    # test with invalid json response
```

Comment:
Will change, originally based this on the way we have written every other test so far, however